### PR TITLE
add slack attribution to keep cards

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner 

@cvializ thank you for the svg help!

### CODE REVIEW

**Comment by camhashemi** [comment]
screen shot 2015-11-19 at 4 52 01 pm
screen shot 2015-11-19 at 4 52 11 pm
screen shot 2015-11-19 at 4 52 22 pm

**Comment by andrewconner** [comment]
Nice! Why does it wrap lines? The twitter one:

**Comment by camhashemi** [comment]
I was going off of https://app.asana.com/0/inbox/32728208409722

no preference, can change it. it cleans up the name row.

File: kifi-backend/modules/shoebox/angular/src/feed/feed.tpl.html [hunk 1]
```diff
@@ -97,6 +97,18 @@ <h4 class="kf-feed-heading">
...
         ></a> &mdash; <span am-time-ago="feedItem.createdAt"></span>
       </h4>
+      <div class="kf-feed-attr">
+        <div class="kf-feed-attr-twitter" ng-if="::feedItem.sourceAttribution.twitter.screenName">
+          <a class="kf-feed-attr-link" href="https://www.twitter.com/{{::feedItem.sourceAttribution.twitter.screenName}}/status/{{::feedItem.sourceAttribution.twitter.idString}}" target="_blank" ng-click="trackTweet()">
+            <span class="kf-keep-twitter-logo svg-twitter-blue"></span>View the tweet
+          </a>
+        </div>
+        <div class="kf-feed-attr-slack" ng-if="::feedItem.sourceAttribution.slack">
+          <a class="kf-feed-attr-link kf-flex-row" href="{{::feedItem.sourceAttribution.slack.message.permalink}}" target="_blank" ng-click="trackSlack()">
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/feed/feed.tpl.html]: `ng-href` is preferred

File: kifi-backend/modules/shoebox/angular/src/feed/feed.tpl.html [hunk 2]
```diff
@@ -97,6 +97,18 @@ <h4 class="kf-feed-heading">
...
+            <span class="svg-slack kf-feed-slack-icon"></span><span>via @{{ feedItem.sourceAttribution.slack.message.username }}</span>
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/feed/feed.tpl.html]: `ng-bind` is preferred

**Comment by cvializ** [comment]
Besides the minor `ng-hrefs` and `ng-binds` instead of `<tag>{{ blah }}</tag>` looks good.

Can't wait to see it : D

**Pull request merged**